rust/common/cookieless/src/hash.rs

```diff
@@ -0,0 +1,195 @@
+use siphasher::sip128::{Hasher128, SipHasher24};
```

Comment:
This file is just moving code out of lib.rs unchanged

Reply:
Didn't review this file, assuming it's exactly the same